test/e2e/smoketest.go

```diff
@@ -2,35 +2,68 @@ package e2e
 
 import (
 	"errors"
+	"fmt"
 	"io/ioutil"
 	"net/http"
 	"strings"
 	"time"
 
+	"github.com/stretchr/testify/require"
 	"github.com/uber/jaeger-client-go/config"
 	"k8s.io/apimachinery/pkg/util/wait"
 )
 
-func SmokeTest(apiTracesEndpoint, collectorEndpoint, serviceName string, interval, timeout time.Duration) error {
+// This version is for the all-in-one image, where query and collector use the same pod
+func SmokeTest(queryPodPrefix, queryPodImageName, serviceName string, interval, timeout time.Duration) {
```

Comment:
Rather than `queryPod...` they should be `allInOnePod....` - to distinguish from the separate query/collector version below. Possibly the function should also be called AllInOneSmokeTest ?